crypto/zk/circuit/circuit.go

```diff
@@ -0,0 +1,263 @@
+package circuit
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"time"
+
+	"go.vocdoni.io/dvote/log"
+)
+
+// By default set the circuit base dir empty which means that the artifacts will
+// be downloaded or loaded from the LocalDir path associated to the circuit config
+var circuitsBaseDir = ""
+var BaseDir = &circuitsBaseDir
```

Comment:
Is this pointer necessary? Why not just declaring BaseDir as string?

Reply:
I thought so, but after some testing I think not. What I was trying to do was to keep the value of the variable between packages, to support that one package could update the value of the variable and ensure that other packages could read the same value. And I thought that the best way to do this would be to use pointers. Example: https://replit.com/@lucasmenendez/testGlobalVars